src/Console/Command/ValidateLayoutFileCommand.php

```diff
@@ -21,7 +22,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $file = $input->getArgument('file');
 
         if (!file_exists($file)) {
-            $output->writeln("<error>The requested file was not found or is not accessible.</error>");
+            $output->writeln('<error>The requested file was not found or is not accessible.</error>');
```

Comment:
Call to method `writeln` from undeclared class `\Symfony\Component\Console\Output\OutputInterface`